src/index.html

```diff
@@ -11,8 +14,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheet with .scss extension should not be linked directly in HTML. Instead, compile it to .css and link the compiled stylesheet.